### ISSUE TITLE
Remember the window size

### ISSUE THREAD

**Issue by novns** [smolBlackCat/progress-tracker#31]
The app always starts as a small square window, only one board link is visible.

You need to change the window size every time to see all boards and all columns in a board.

Remembering the last windows size would be good.

**Comment by leandromattioli** [comment]
Well, starting the application with a 400x400 window indeed isn't a good option. Until your suggestion ("remembering" window size) is properly implemented, I suggest starting the main window maximized (it is already better than starting too small).

Adding the line this->maximize() at the bottom of the constructor is sufficient for this matter.

```
diff --git a/src/window.cpp b/src/window.cpp
index 37ee632..20b8009 100644
--- a/src/window.cpp
+++ b/src/window.cpp
@@ -154,6 +154,8 @@ ProgressWindow::ProgressWindow(BaseObjectType* cobject,
     delete_boards_bar.set_margin_bottom(10);
     app_overlay_p->add_overlay(delete_boards_bar);
     app_stack_p->add(board_widget, "board-page");
+
+    this->maximize();
 }
 
 ProgressWindow::~ProgressWindow() {
```

**Comment by smolBlackCat** [comment]
> The app always starts as a small square window, only one board link is visible.

Yeah, I'm aware of this pesky situation. I was planning on adding this and another bunch of new features in the next major release. But if this is something that may be inconvenient for some users, we can add a temporary solution, like the one mentioned by @leandromattioli

**Comment by smolBlackCat** [comment]
This issue is now solved in Progress 2.0 #26
